### PR TITLE
Fix replica count

### PR DESCRIPTION
**Pull request by kgroschoff**
### What this PR does / why we need it
Fix replica count, as it was automatically reset to default after a few seconds. 

### Release note

```release-note
Fixed replica count, as it was automatically reset to default after a few seconds.
```

### CODE REVIEW

File: src/app/node-data/component.ts [hunk 1]
```diff
@@ -176,8 +176,10 @@ export class NodeDataComponent extends BaseFormValidator implements OnInit, OnDe
       .subscribe(_ => (this._nodeDataService.operatingSystemSpec = this._getOperatingSystemSpec()));
 
     this._settingsService.adminSettings.pipe(takeUntil(this._unsubscribe)).subscribe(settings => {
```
Comment (maciaszczykm) [inline src/app/node-data/component.ts:178]: If we would change this pipe to `take(1)` then the bug should be gone. The service is responding very quickly as admin settings are cached but the problem is that there is refresh happening sometimes.

Comment (maciaszczykm) [reply src/app/node-data/component.ts:178]: The initialization should happen only once anyways so there is no point of keeping this subscription open.

Comment (kgroschoff) [reply src/app/node-data/component.ts]: ah you're right. Always forget `take(1)` :see_no_evil:
Changed 👍

File: src/app/node-data/component.ts [hunk 2]
```diff
@@ -175,10 +175,13 @@ export class NodeDataComponent extends BaseFormValidator implements OnInit, OnDe
       .pipe(takeUntil(this._unsubscribe))
       .subscribe(_ => (this._nodeDataService.operatingSystemSpec = this._getOperatingSystemSpec()));
 
-    this._settingsService.adminSettings.pipe(takeUntil(this._unsubscribe)).subscribe(settings => {
-      const replicas = this.dialogEditMode ? this._nodeDataService.nodeData.count : settings.defaultNodeCount;
-      this.form.get(Controls.Count).setValue(replicas);
-    });
+    this._settingsService.adminSettings
+      .pipe(take(1))
+      .pipe(takeUntil(this._unsubscribe))
```
Comment (maciaszczykm) [inline src/app/node-data/component.ts:180]: Last thing would be to remove this line as usually we don't keep both together. There is almost no chance to fire unsubscribe before admin settings cache would be returned.

Comment (kgroschoff) [reply src/app/node-data/component.ts]: done :)

**Pull request merged**